core/templates/components/question-directives/question-misconception-editor/question-misconception-editor.component.spec.ts

```diff
@@ -29,10 +28,10 @@ import { RuleObjectFactory } from 'domain/exploration/RuleObjectFactory';
 import { SolutionValidityService } from
   'pages/exploration-editor-page/editor-tab/services/solution-validity.service';
 import { StateEditorService } from
+  /* eslint-disable-next-line max-len */
```

Comment:
Same thing, here and elsewhere, you don't need the disabling I think. You could perhaps put everything on the same line as the import if dropping it as-is doesn't work.

That said, for this one and the files below, it's optional. Happy to leave it as you have it, too, though it might be nice to move more eslint-disables so it doesn't become too much of a "standard pattern".